.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,71 @@
+# Contributing to dfeshiny
+
+Try and make use of the [usethis](https://usethis.r-lib.org/) package wherever possible.
+
+When you initially clone the package, the first thing you'll need to do is install [devtools](https://devtools.r-lib.org/):
+
+```
+install.packages("devtools")
+```
+
+Then to load in the package in its current form:
+
+```
+devtools::load_all()
+```
+
+## Adding a package/dependency
+
+`usethis::use_package(<package_name>)`
+
+Note that when adding a function from another package into one of the dfeshiny functions you will need to explicitly state the package in the function call, e.g.:
+
+```package::function()```
+
+Alternatively, if there's a lot of uses of a single function within one of our R scripts, you can call that function once at the top of the R script, e.g:
+
+```
+@' importFrom package function
+```
+
+For more information see the [roxygen2 documentation on declaring dependencies](https://roxygen2.r-lib.org/articles/namespace.html).
+
+## Creating a new function script
+
+`usethis::use_r(name = <script_name>)`
+
+This will create a new script within the package R/ folder.
+
+## Creating a new function test script
+
+`usethis::use_test(name = <script_name>)`
+
+This will create a new test script within the package testthat/ folder.
```

Comment:
Maybe worth highlighting this is a blank script, it doesn't do anything clever like create tests for you.

Reply:
Fixed